client/post-editor/editor-action-bar/index.jsx

```diff
@@ -67,7 +67,7 @@ export default React.createClass( {
 
 		return (
 			<div className="editor-action-bar">
-				<div className="editor-action-bar__first-group">
+				<div className="editor-action-bar__cell editor-action-bar__cell-left">
```

Comment:
The column-specific classnames seem to fall well under our concept of a modifier:

https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#csssass-coding-guidelines

In other words, I think it's reasonable to change them to apply as extensions of the base class:

```scss
.editor-action-bar__cell {
	width: 33%;

	&.is-center {
		text-align: center;
	}

	&.is-right {
		text-align: right;
	}
}
```